Tests/Configuration/configurationGenerationSpec.swift

```diff
@@ -19,8 +19,8 @@ class ConfigurationGenerationSpec: QuickSpec {
                 context("when there isn't an Xcode project in the project directory") {
                     it("prefills a configuration to use the `swift test` command") {
                         let projectDirectoryContents = ["/some/path/Package.swift",
-                                                        "/some/path/main.swift",]
-                        
+                                                        "/some/path/main.swift" ]
```

Comment:
Not loving this style. I'd prefer this:

```swift
let foo = [
    "hello",
    "world",
]
```

Reply:
I blame swiftlint for it :)

Reply:
I propose adding this to .swiftlint.yml:

```yml
trailing_comma:
  mandatory_comma: true
```